utils/fusesoc_get_sources.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
```

Comment:
Should this move upstream into fusesoc in the future?

Reply:
Definitely! It would be a great feature! 

On the fusesoc side, it can be done using internal data structures. 
In this small example, I think It is better to parse the Edalize YAML produced by fusesoc. Otherwise, we will have to maintain another forked tool. We can independently prepare PR to fusesoc and change the example when it is merged.